amethyst_renderer/src/lib.rs

```diff
@@ -91,6 +91,7 @@ pub use formats::{
     GraphicsPrefab, ImageData, JpgFormat, MaterialPrefab, MeshCreator, MeshData,
     ObjFormat, PngFormat, TextureData, TextureFormat, TextureMetadata, TexturePrefab,
 };
+pub use gfx::texture::{FilterMethod, SamplerInfo, WrapMode};
```

Comment:
I don't think it's a good idea to expose gfx types. As @jojolepro mentioned, it will be removed in the future (or at least changed to hal) so these will undoubtedly break in the future for some people.

Reply:
Do you have an alternative?

Reply:
I would say make wrapper types that are essentially the same and then convert them to gfx types when needed.

Reply:
Then what is the difference?

Reply:
That they won't break when we change the renderer?

Reply:
Wrapping them would make it break anyway.

Reply:
No it wouldn't. You just change the mapping between the types.

Reply:
Okay, then just re-exporting also doesn't make it break. You just have to fit to the interface.

Reply:
Make. Wrapper. Types. To. Not. Break. User. Code. :P

Normally we don't care to break the user api, but we don't want to force people to import gfx just to break it in their face in the next version.

Reply:
But it breaks user code either way...